_docs/tutorials/how-to-use-jwt-in-sn-client-js.md

```diff
@@ -0,0 +1,332 @@
+---
+title:  "JWT Authentication in sn-client-js"
+source_url: 'https://github.com/SenseNet/sensenet.github.io/blob/master/_docs/tutorials/how-to-use-jwt-in-sn-client-js.md'
+category: Tutorials
+version: v7.0
+tags: [sn7, sn-client-js, jwt, authentication, frontend]
+---
+
+# JWT Authentication in sn-client-js
+
+With version 7.0 sensenet ECM has a powerful feature to authenticate users using [JSON Web Tokens](/docs/web-token-authentication), which is supported by [sn-client-js](https://www.npmjs.com/package/sn-client-js) an [sn-redux](https://www.npmjs.com/package/sn-redux) as well. 
+
+
+> **What is a JSON Web Token?** JSON Web Token is an open standard ([RFC 7519](https://tools.ietf.org/html/rfc7519)) that defines a compact, self-contained way for securely transmitting data between parties as a JSON object. The information can be verified and trusted because it's signed digitally. The most common scenarios of JWT are *authentication* and *secure data exchange*.
+
+## Backend prerequisites
+In order to use JWT you need a preconfigured installation of **sensenet ECM Services** backend.
+
+You can start by creating a new ASP.NET Web Application. Grab the latest **[Sensenet.Services.Install](http://www.nuget.org/packages/SenseNet.Services.Install/)** package from NuGet and follow the installation instructions.
+
+In order to get JWT work you have to update your web.config, filling the *SymmetricKeySecret* variable with a 16-64 character length random string.
+
+```xml
```

Comment:
This section should be added automatically when using Sensenet.Services >=7.0.0-beta2 with an empty *SymmetricKeySecret* value